tkg/constants/files.go

```diff
@@ -39,3 +39,67 @@ const (
 
 	TKGPackageValuesFile = "tkgpackagevalues.yaml"
 )
+
+// Config content
+const (
```

Comment:
Why do we need a const here?

Reply:
Separate const with different usage

Reply:
```
// ConfigFilePermissions defines the permissions of the config file
const (
	ConfigFilePermissions       = 0o600
	DefaultDirectoryPermissions = 0o700
)

// File name related constants
const (

```

Reply:
Or you want me to put the content in a real fie

Reply:
Ok I moved the content to providers folder